src/main/java/org/jenkinsci/plugins/proccleaner/PreBuildCleanup.java

```diff
@@ -69,6 +68,7 @@ public Environment setUp(AbstractBuild build, Launcher launcher,
     }
 
     @Extension
+    @SuppressWarnings("deprecation")
```

Comment:
Can it be moved to more focused scope than the whole class? Not sure what is the deprecated API here...